images/npm-release/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:12.16.2-alpine3.11
+
+RUN apk add --no-cache jq
```

Comment:
Should we just run the `apk` command to add `jq` as part of the job's tasks and not deal with another image that we have build, push, and version ourselves?
What do you think?

Reply:
Definitely a valid suggestion.  As lightweight as `jq` is, I still think there's benefit in avoiding the tax of installation at runtime, which would occur when/if added to the job's tasks.  Also, having the foundation of a managed image here allows us to add additional utilities down the line, perhaps paired with enhancements to this helper class (but this argument is admittedly hypothetical.)

Since we have the CI framework already for build/publish of other images (kind image for example), and it is built in such a way that the tax is low to add new images to this CI process, I think I'd favor keeping this as is.

Reply:
That makes sense - marking this as resolved, thanks!